test/module/irohad/network/block_loader_test.cpp

```diff
@@ -52,15 +60,30 @@ class BlockLoaderTest : public testing::Test {
     builder.RegisterService(service.get());
     server = builder.BuildAndStart();
 
-    peer = Peer();
+    Peer peer;
     peer.address = "0.0.0.0:" + std::to_string(port);
+    std::copy_n(
+        peer_key.blob().begin(), peer.pubkey.size(), peer.pubkey.begin());
     peers.push_back(peer);
 
     ASSERT_TRUE(server);
     ASSERT_NE(port, 0);
   }
 
-  Peer peer;
+  auto base() const {
+    constexpr auto kTotal = (1 << 5) - 1;
```

Comment:
Not clear. What does it mean?

Reply:
Exactly what does TOTAL mean in builder, but it's private